third_party/conan/configs/linux/settings.yml

```diff
@@ -69,7 +69,7 @@ compiler:
     clang:
         version: ["3.3", "3.4", "3.5", "3.6", "3.7", "3.8", "3.9", "4.0",
                   "5.0", "6.0", "7.0", "7.1",
-                  "8", "9", "10"]
+                  "8", "9", "10", "11", "12"]
```

Comment:
do we need profiles for these?

Reply:
Eventually yes. But for current use-case, no.

I'm a bit reluctant with adding the profiles without having at least once compiled the whole project with these compilers. I have a branch which is adding clang-10 profiles, but I haven't tried that yet. But I guess this will follow in a future PR. clang-11 is not released yet. Release is scheduled for tomorrow. I might add support for 10 and 11 in one PR.

We only need 12 because oss-fuzz uses clang-master which currently identifies as clang-12.